projects/igniteui-angular/src/lib/grids/api.service.ts

```diff
@@ -143,6 +143,8 @@ export class GridBaseAPIService <T extends IgxGridBaseDirective & GridType> {
         cell.editValue = value;
 
         const args = cell.createEditEventArgs();
+        args.cell = this.grid.getCellByColumn(args.cellID.rowIndex, args.column.field);
+        args.owner = this.grid;
```

Comment:
Maybe it is good idea this code snippet to be part of createEditEventArgs() method?